src/libraries/System.Resources.Extensions/tests/BinaryResourceWriterUnitTest.cs

```diff
@@ -500,6 +501,50 @@ public static void EmbeddedResourcesAreUpToDate()
             }
         }
 
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsBinaryFormatterSupported))]
+        /// <summary>
+        /// This test has multiple threads simultaneously loop over the keys of a moderately-sized resx using
+        /// <see cref="ResourceManager"/> and call <see cref="ResourceManager.GetString(string)"/> for each key.
+        /// This has historically been prone to thread-safety bugs because of the shared cache state and internal
+        /// method calls between RuntimeResourceSet and <see cref="DeserializingResourceReader"/>.
+        ///
+        /// Running with <paramref name="useEnumeratorEntry"/> TRUE replicates https://github.com/dotnet/runtime/issues/74868,
+        /// while running with FALSE replicates the error from https://github.com/dotnet/runtime/issues/74052.
+        /// </summary>
+        /// <param name="useEnumeratorEntry">
+        /// Whether to use <see cref="IDictionaryEnumerator.Entry"/> vs. <see cref="IDictionaryEnumerator.Key"/> when enumerating;
+        /// these follow fairly different code paths.
+        /// </param>]
+        [InlineData(false)]
+        [InlineData(true)]
```

Comment:
```suggestion
        /// <summary>
        /// This test has multiple threads simultaneously loop over the keys of a moderately-sized resx using
        /// <see cref="ResourceManager"/> and call <see cref="ResourceManager.GetString(string)"/> for each key.
        /// This has historically been prone to thread-safety bugs because of the shared cache state and internal
        /// method calls between RuntimeResourceSet and <see cref="DeserializingResourceReader"/>.
        ///
        /// Running with <paramref name="useEnumeratorEntry"/> TRUE replicates https://github.com/dotnet/runtime/issues/74868,
        /// while running with FALSE replicates the error from https://github.com/dotnet/runtime/issues/74052.
        /// </summary>
        /// <param name="useEnumeratorEntry">
        /// Whether to use <see cref="IDictionaryEnumerator.Entry"/> vs. <see cref="IDictionaryEnumerator.Key"/> when enumerating;
        /// these follow fairly different code paths.
        /// </param>]
        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsBinaryFormatterSupported))]
        [InlineData(false)]
        [InlineData(true)]
```

Reply:
Moved the attribute.